src/qiskit_sphinx_theme/furo_py/__init__.py

```diff
@@ -0,0 +1,440 @@
+# This file is vendored from Furo (created by Pradyun Gedam) and used under the MIT license.
+#
+# When making changes, surround it with `QISKIT CHANGE: start` and `QISKIT CHANGE: end` comments.
+# We also add MyPy type ignores.
```

Comment:
This file is copied exactly, other than the parts where I put `QISKIT CHANGE: start`.